client/landing/gutenboarding/components/login-form/index.tsx

```diff
@@ -59,18 +85,32 @@ const LoginForm = () => {
 	// todo: may need to be updated as more states are handled
 	const shouldShowPasswordField = loginFlowState === 'ENTER_PASSWORD';
 
+	const changeEmailOrUsernameLinkText = includes( usernameOrEmailVal, '@' )
+		? NO__( 'Change Email Address' )
+		: NO__( 'Change Username' );
+
 	return (
 		<Modal
 			className="login-form"
 			isDismissible={ false }
 			title={ NO__( 'Log in to save your changes' ) }
-			onRequestClose={ handleClose }
+			onRequestClose={ closeModal }
 		>
 			<form onSubmit={ handleLogin }>
+				{ shouldShowPasswordField && (
+					<Button
+						className="login-form__change-username"
+						isLink={ true }
+						onClick={ changeEmailAddress }
+					>
+						<Icon icon="arrow-left-alt" size={ 18 } />
+						{ changeEmailOrUsernameLinkText }
+					</Button>
+				) }
 				<TextControl
-					label={ NO__( 'Email Address or Username' ) }
+					label={ ! shouldShowPasswordField ? NO__( 'Email Address or Username' ) : '' }
 					value={ usernameOrEmailVal }
-					// disabled={ isLoading }
+					disabled={ shouldShowPasswordField }
```

Comment:
I can't change my username if the password field is showing. Will this be in the final version?

Reply:
that's the plan at the moment, it makes the implementation quite a bit easier/more elegant and matches what's in production now. on `/log-in`. I've updated the CSS so that the disabled state is now visible in chrome!